src/test/java/com/conveyal/r5/analyst/network/SimpsonDesertTests.java

```diff
@@ -58,18 +59,16 @@ public void testGridScheduled () throws Exception {
         // Write travel times to Geotiff for debugging visualization in desktop GIS:
         // toGeotiff(oneOriginResult, task);
 
-        int destination = gridLayout.pointIndex(task, 40, 40);
-        int[] travelTimePercentiles = oneOriginResult.travelTimes.getTarget(destination);
-
         // Transit takes 30 seconds per block. Mean wait time is 10 minutes. Any trip takes one transfer.
         // 20+20 blocks at 30 seconds each = 20 minutes. Two waits at 0-20 minutes each, mean is 20 minutes.
-        // Board slack is 2 minutes. With pure frequency routes total should be 24 to 64 minutes, median 44.
-        // However, these are not pure frequencies, but synchronized such that the transfer wait is always 10 minutes.
-        // So scheduled range is expected to be 2 minutes slack, 0-20 minutes wait, 10 minutes ride, 10 minutes wait,
-        // 10 minutes ride, giving 32 to 52 minutes.
-        // Maybe codify this estimation logic as a TravelTimeEstimate.waitWithHeadaway(20) etc.
-        DistributionTester.assertUniformlyDistributed(travelTimePercentiles, 32, 52);
-
+        // Board slack is 1 minute. These are not pure frequencies, but synchronized such that the transfer wait is
+        // always 10 minutes. So scheduled range is expected to be 1 minute slack, 0-20 minutes wait, 10 minutes ride,
+        // 10 minutes wait, 10 minutes ride, giving 31 to 51 minutes.
+        // This estimation logic could be better codified as something like TravelTimeEstimate.waitWithHeadaway(20) etc.
+
+        // TODO For some reason observed is off by 1 minute, figure out why.
```

Comment:
Is board slack being applied at the second (transfer) boarding?

Reply:
I would not expect it to be applied, as the purpose of slack is to allow at least N minutes (currently set to 1) between arrival at the transit stop and departure of the vehicle, not to add 1 minute to every boarding time. In a case where the transfer target vehicle always departs 10 minutes after the passenger arrives at the stop, the fixed 10 minute wait exceeds the 1 minute slack, so I would expect the total wait to always be 10 minutes. However `FastRaptorWorker#BOARD_SLACK_SECONDS` seems to only be used in the multi-criteria RAPTOR code, and there's another constant `FastRaptorWorker#MINIMUM_BOARD_WAIT_SEC` which serves a very similar purpose and comments advising that these two be merged. Uses of this `MINIMUM_BOARD_WAIT_SEC` imply that it would just search for any departure more than one minute later (so finding one 10 minutes later rather than 11). I will keep digging into this though to see if there's anywhere we'd just be adding the slack.

Reply:
I moved some constants and updated some comments to make things a little clearer. Board slack (actually MINIMUM_BOARD_WAIT_SEC) seems to work as I expected, establishing a minimum wait but not always introducing an additional wait. My best guess now for the source of the extra minute is some kind of edge effect due to the fact that we're binning times into one-minute bins, and something may be pushing the travel time over the edge into the next bin. I'll have to look at the travel time in seconds instead of minutes to confirm this.